database/generate-data.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs');
+const data = require('./data.js');
+// generate data
+// write to csv file
+
+const generateData = (field, fileName) => {
+  // i: string, and string
+  // o: side effect, writes field content to fileName
+
+  let info = field.split('\n').join(',');
+  fs.writeFile(`${fileName}.txt`, info, (err) => {
+    if (err) {
+      console.log('there was an error!');
+    } else {
+      console.log('write was successful!');
+    }
+  })
+
+};
+
+const getRandomIndex = (num) => {
+  // i: number
+  // o: number, random between 0 and num
+
+  function getRandomIntInclusive(min, max) {
```

Comment:
Try refactoring to ES6 style for consistency

Reply:
Also, does this need to be an inner function? Maybe you can refactor so there is no inner function